addons/web/models/ir_http.py

```diff
@@ -101,6 +101,7 @@ def get_frontend_session_info(self):
             'is_system': request.session.uid and self.env.user._is_system() or False,
             'is_website_user': request.session.uid and self.env.user._is_public() or False,
             'user_id': request.session.uid and self.env.user.id or False,
+            'uid': request.session.uid and self.env.user.id or False,
```

Comment:
@qsm-odoo FYI, see commit message, that mystery is now solved :o 
Would you backport it? I think it is fine in 15.0 only?

Reply:
I would definitely not backport it, I am even unsure it should be be added in 15.0, especially since there is a no-rpc workaround with `session.user_id`... maybe we could just patch the `user_has_group` function in stable? :grimacing: 

AFAICR `session.uid` has always been falsy in frontend environments (never knew if on purpose and never actually needed it anyway), I would not be surprised some code is doing `if (!session.uid) {` instead of `if (session.is_frontend) {` TBH :grimacing:

Reply:
Valid point. Better safe than sorry.